doc/policies/conditions.rst

```diff
@@ -0,0 +1,128 @@
+.. _policy_conditions:
+
+Policy conditions
+-----------------
+
+Since privacyIDEA 3.1, *policy conditions* allow to define more advanced rules
+for policy matching, i.e. for determining which policies are valid for a
+specific request.
+
+Conditions can be added to a policy via the WebUI. In order for a policy to
+take effect during the processing of a request, the request has to match not
+only the ordinary policy attributes (see :ref:`policies`), but also *all*
+additionally defined conditions that are currently active. If no active
+conditions are defined, only the ordinary policy attributes are taken into
+account.
+
+Each policy condition performs a comparison of two values. The left value is
+taken from the current request. The comparison operator (called *Comparator*)
+and the right value are entered in the policy definition. Each condition
+consists of five parts:
+
+ * ``Active`` determines if the condition is currently active.
+ * ``Section`` refers to an aspect of the incoming request on which the condition is formulated.
+   The available sections are predefined, see `Sections`_.
+ * The meaning of ``Key`` depends on the chosen ``Section``. Typically, it determines the exact property
+   of the incoming request on which the condition is formulated.
+ * ``Comparator`` defines the comparison to be performed. The available comparators are predefined, see `Comparators`_.
+ * ``Value`` determines the value the property should be compared against.
+
+Sections
+~~~~~~~~
+
+``userinfo``
+^^^^^^^^^^^^
+
+The validity of a policy condition with section ``userinfo`` is determined as follows:
+
+* privacyIDEA retrieves the userinfo of the currently managed user. These are the user attributes as they are
+  determined by the respective resolver. This is configured via the attribute mappings of resolvers
+  (see :ref:`useridresolvers`).
+* Then, it retrieves the userinfo attribute given by ``Key``
+* Finally, it uses the ``Comparator`` to compare the contents of the userinfo attribute with the given ``Value``.
+  The result of the comparison determines if the request matches the condition or not.
+
+.. note:: There are situations in which the currently managed user
+    cannot be determined.  If privacyIDEA encounters a policy with ``userinfo``
```

Comment:
Shouldn't the following lines of a ``note`` section be indented by 3 spaces?

Reply:
Heh, I guess so :) Done in 4c03d0e7